website/project/views/node.py

```diff
@@ -249,7 +249,8 @@ def node_setting(auth, node, **kwargs):
 
     #check institutions:
     try:
-        inst = Institution.find_one(Q('email_domain', 'eq', auth.user.username.split('@')[1]))
+        email_domains = [email.split('@')[1] for email in auth.user.emails]
+        inst = Institution.find_one(Q('email_domain', 'in', email_domains))
```

Comment:
OK. As suggested in a previous comment, `email_domain` should probably be renamed to `email_domains` to make it clear that it's a list.